docs/rails.md

```diff
@@ -75,3 +81,6 @@ Rails.application.routes.draw do
   end
 end
 ```
+
+<!-- Bibliography -->
+[12-factor config]: https://12factor.net/config
```

Comment:
I think most changes in this file are out of scope of this PR, and that we
actually lost some information, and added a little confusion.
I would extract them in a new PR (or multiple ones) so that we can discuss
them.